packages/flutter/lib/src/painting/inline_span.dart

```diff
@@ -100,6 +100,42 @@ class InlineSpanSemanticsInformation {
   String toString() => '${objectRuntimeType(this, 'InlineSpanSemanticsInformation')}{text: $text, semanticsLabel: $semanticsLabel, recognizer: $recognizer}';
 }
 
+/// Combines _semanticsInfo entries where permissible.
+///
+/// Consecutive inline spans can be combined if their
+/// [InlineSpanSemanticsInformation.requiresOwnNode] return false.
+List<InlineSpanSemanticsInformation> combineSemanticsInfo(List<InlineSpanSemanticsInformation> infoList) {
```

Comment:
Has the logic here changed from the original _combineSemanticsInfo, or was it just moved?